src/data/move-attrs/override-move-effect-attr.ts

```diff
@@ -22,7 +22,7 @@ export class OverrideMoveEffectAttr extends MoveAttr {
    */
   override apply(
     _user: Pokemon,
-    _target: Pokemon,
+    _target: Pokemon | null,
```

Comment:
Q: Wasn't the plan to move to a consistent `_target?: Pokemon` or if `move` etc. is required -> `_target: Pokemon | undefined`?